packages/cardano-services/src/Http/HttpServer.ts

```diff
@@ -136,9 +122,33 @@ export class HttpServer extends RunnableModule {
     };
 
     const handlers = {
-      health: healthCheckHandler(() => 200),
+      health: async (req: express.Request, res: express.Response) => {
+        this.logger.debug('/health', { ip: req.ip });
+        let health: ServicesHealthCheckResponse | Error['message'];
+        try {
+          health = await this.#getServicesHealth();
+          return HttpServer.sendJSON(res, health, 200);
+        } catch (error) {
+          this.logger.error(error);
+          return HttpServer.sendJSON(res, new ProviderError(ProviderFailure.Unhealthy, error), 500);
+        }
+      },
+      live: async (req: express.Request, res: express.Response) => {
+        this.logger.debug('/live', { ip: req.ip });
+        return res.sendStatus(200);
+      },
       meta: serverMetadataHandler,
```

Comment:
Wait a second, this does not do much, in definitions we should not replace liveliness probe because this acts more like startup probe

Reply:
We have `/live` and `/ready`, with the former being somewhat of a startup test. Is that not what the liveliness probe needs?

Reply:
Short summary is:
liveliness probe - assesses if an application is healthy
startup probe - assesses if an application has started
readiness probe - assesses if an application is ready to accept connections